### DISCUSSION TITLE
About difference of graphql and rest Api

### DISCUSSION THREAD

**Discussion by BigMoney031026** [community/community#37113] [Other Feature Feedback, Questions, & Ideas] [answered]
I want to know about difference of graphql and rest Api.
if some one know about this let me know

**Comment by newstable** [comment, marked as answer]
The core difference between GraphQL and REST APIs is that GraphQL is a specification, a query language, while REST is an architectural concept for network-based software. Note: This article is mostly server-side related. GraphQL is gaining momentum as a successor to REST APIs

**Reply by BigMoney031026** [reply]
thank you for your answer.

now I can understand about difference of graphql and rest api

**Comment by ossanna16** [comment]
Hi there @BigMoney031026 and welcome to our community 😄. Thanks so much for asking a great question!

**Reply by BigMoney031026** [reply]
Hello @ossanna16
May my question help you a bit?

**Reply by ossanna16** [reply]
Yes of course, every question always helps other people. There's always someone who has the same question as you :)

**Comment by Khushboqof** [comment]
Here is the important difference between GraphQL and REST API. GraphQL is an application layer server-side technology which is developed by Facebook for executing queries with existing data. REST is a software architectural style that defines a set of constraints for creating Web services

**Reply by BigMoney031026** [reply]
Thank you for your answer
now I can understand about difference between Graphql and rest api

**Comment by seagodcoder** [comment]
As I hope you can see from this article, GraphQL has a number of advantages over REST in many areas. It might take more time to set up GraphQL initially, but there are many scaffolders which make that job easier. And even if it takes more time in the beginning, it gives you long term advantages and is totally worth it.

**Comment by 0xi3gold** [comment]
You can learn more deeply by checking this article.
https://www.guru99.com/graphql-vs-rest-apis.html
Thanks.
